posts/2020/04/introducing-ndindex-a-python-library-for-manipulating-indices-of-ndarrays.markdown

```diff
@@ -0,0 +1,325 @@
+<!--
+.. title: Introducing ndindex, a Python library for manipulating indices of ndarrays
+.. slug: introducing-ndindex-a-python-library-for-manipulating-indices-of-ndarrays
+.. date: 2020-04-09 15:42:24 UTC-05:00
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
```

Comment:
Is there not an author entry?

Reply:
Oh yeah. I guess it doesn't ask you for it. I added some metadata. We will also need to update the date whenever this gets merged.